content/en/docs/tasks/configure-pod-container/configure-service-account.md

```diff
@@ -296,6 +296,59 @@ token available to the pod at a configurable file path, and refresh the token as
 
 The application is responsible for reloading the token when it rotates. Periodic reloading (e.g. once every 5 minutes) is sufficient for most usecases.
 
+## Service Account Issuer Discovery
+
+{{< feature-state for_k8s_version="v1.18" state="alpha" >}}
+
+{{< note >}}
+The Service Account Issuer Discovery feature is __alpha__ in 1.18 and enabled by
```

Comment:
This is repeating the feature-state line. I would tell readers how to enable the feature gate (what component?) and omit the repetition.

Reply:
Done.